sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -629,27 +629,27 @@ public virtual Pageable<ConfigurationSetting> GetConfigurationSettings(SettingSe
         }
 
         /// <summary>
-        /// Retrieves the different revisions of a specific <see cref="ConfigurationSetting"/> with the specified <paramref name="key"/> and <paramref name="label"/>.
+        /// Retrieves the different revisions of a specific <see cref="ConfigurationSetting"/> with the specified <paramref name="keyFilter"/> and <paramref name="labelFilter"/>.
```

Comment:
This comment (and param comments) should be updated to reflect what the filter is doing -- it's no longer retrieving revisions for a single Config Setting.

Reply:
Good catch